src/main/java/uk/gov/ons/fwmt/job_service/service/impl/totalmobile/TMServiceImpl.java

```diff
@@ -133,15 +184,17 @@ protected String lookupSOAPAction(Class<?> cl) {
   }
 
   public <I, O> O send(I message) {
+    log.debug("send: Began sending message of class {}", message.getClass().getSimpleName());
```

Comment:
Can we log the request and response to TM. This would be useful in case we are doing monitoring.

Reply:
Info Level is fine. why TRACE level?

Reply:
Simply because of the size of the output. I'll change it to INFO, if you're sure that's okay.